### ISSUE TITLE
[Bug/Webapp] Image upload wrongly results in error 

### ISSUE THREAD

**Issue by gadilashashank** [devRant/devRant#435]
When attaching a .png image of <10MB size(177 KB) and posting on the webapp an error was raised twice before posting which said something like "Error in processing image..." The image was not changed but twice the error was raised but not the third time. I'm not sure what more info I can give...

**Comment by AlgoRythm-Dylan** [comment]
Will probably need more info to work on a solution. I assume this was the web app, since you mentioned Firefox. Is your connection slow? What version of Firefox? Has the issue persisted (try it now)?

**Comment by gadilashashank** [comment]
Yes this issue occurred in the webapp. Firefox version is 61.0b7 (64-bit Ubuntu client). No I can assure the internet was not slow. As for the persistence I haven't checked that and I will come back on that..

**Comment by gadilashashank** [comment]
I think this was a random one time issue because one of my recent rants where I posted an image from the webapp did not cause the said error
